modules/get-started/pages/licensing/overview.adoc

```diff
@@ -159,6 +159,10 @@ Redpanda Console offers two methods for applying or updating a license, dependin
 
 - If Redpanda Console is connected to a Redpanda cluster, you can xref:console:ui/add-license.adoc[upload a license through the Redpanda Console UI]. This method allows you to manage and update licenses for both Redpanda Console and the connected Redpanda cluster.
 
+== Manage licenses for Redpanda Connect
+
+Redpanda Connect offers multiple ways to apply or update your license. See xref:redpanda-connect:get-started:licensing.adoc[Enterprise Licensing].
```

Comment:
This link text it a little generic. Could we link to the Apply a license key to Redpanda Connect section and make that the link text?